NSS-Day-Cohort-53/giffygram-rantingtyras
# Src responsive

```text
#### Changes Made
1. Modified file `Navbar.js and navigation.css` to include `responsive hamburger menu` and functionality.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout src-responsive
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > try resizing the width and opening the menu
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
